src/types/types.js

```diff
@@ -359,7 +359,8 @@ export type EdgeTransaction = {
     +uniqueIdentifier?: string
   }>,
   swapData?: EdgeTxSwap,
-  txSecret?: string, // Monero decryption key
+  txSecret?: string, // Monero decryption key,
```

Comment:
Oops, added an extra `,` here on the txSecret line. Also, can you move `deviceDescription` right underneath `// Spend-specific metadata:`.